src/vmm/src/vmm_config/mmds.rs

```diff
@@ -18,6 +18,8 @@ pub struct MmdsConfig {
     pub network_interfaces: Vec<String>,
     /// MMDS IPv4 configured address.
     pub ipv4_address: Option<Ipv4Addr>,
+    /// Compatibility with EC2 IMDS.
+    pub imds_compat: Option<bool>,
```

Comment:
should we just have it `bool` and default to `false`?
```suggestion
    #[serde(default = false)]
    pub imds_compat: bool,
```